wagtail/admin/templates/wagtailadmin/generic/chooser/results.html

```diff
@@ -1,4 +1,28 @@
 {% load i18n wagtailadmin_tags %}
 
-{% component table %}
-{% include "wagtailadmin/shared/pagination_nav.html" with items=results linkurl=results_url_name %}
+{% if results %}
+    {% if is_searching %}
+        {% block search_results_count %}
+            <h2 role="alert">
```

Comment:
Curious as to why this would be a `h2`? The other branches are all `p` tags. 
Also I am pretty sure the role here will override the implicit heading role so the h2 is not doing anything. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role

Reply:
Standard answer for all the front-end code here: I'm just copying what the existing choosers already do :-)

https://github.com/wagtail/wagtail/blob/0705f1d76556891a9c9551b164de4c32ccefcd7a/wagtail/documents/templates/wagtaildocs/chooser/results.html#L4

I'm approaching these PRs as a pure refactoring exercise with as few functional changes as possible - once we have the choosers running on a shared codebase I'm sure there'll be plenty of opportunity for cleaning up this sort of legacy code in one swoop.

Reply:
no problems - I will raise a new issue once this is merged in for this issue.